scripts/create-package.sh

```diff
@@ -0,0 +1,243 @@
+#!/bin/sh
+
+set -x
+
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Copyright (C) 2020-2021  Sherman Perry
+
+usage()
+{
+    printf "%s\n" "\
+    Usage: -p <platform> [-h -c <commit> -b <branch> -r <repo> -v <version>] BUILD_DIR PACKAGE_DIR
+    Where:
+        -h          : Show this help message
+        -p          : Buld platform. Can be one of linux, macos, msys2, msvc
+        -c          : Git commit
+        -b          : Git branch
+        -r          : Git repository
+        -v          : dosbox-staging version
+        BUILD_DIR   : Meson build directory
+        PACKAGE_DIR : Package directory
```

Comment:
It would be nice to require that PACKAGE_DIR be either empty (or not exist).
At least, this is what we expect. 
This would also prevent clobbering existing files or accidentally "polluting" a directory that wasn't intended -- say, a user who sets the package_dir to their home folder (which would make a mess). 

If we later find some users want this behavior, then we add a `-f` (force) flag.